File: Seminar/src/main/java/org/sopt/Seminar/domain/category/repository/CategoryRepository.java
package org.sopt.Seminar.domain.category.repository;

import org.sopt.Seminar.domain.category.model.Category;
import org.sopt.Seminar.domain.category.model.CategoryId;
import org.springframework.data.jpa.repository.JpaRepository;

public interface CategoryRepository extends JpaRepository<Category, CategoryId> {
}

File: kopring/week2/kotlin-semina/src/main/kotlin/org/sopt/kotlinsemina/domain/category/repository/CategoryRepository.kt
package org.sopt.kotlinsemina.domain.category.repository

import org.sopt.kotlinsemina.domain.category.model.Category
import org.sopt.kotlinsemina.domain.category.model.CategoryId
import org.springframework.data.jpa.repository.JpaRepository

interface CategoryRepository : JpaRepository<Category, CategoryId> {
}